0.1/files/main.sh

```diff
@@ -9,13 +9,14 @@ S3CMD_PATH=/opt/s3cmd/s3cmd
 # Check for required parameters
```

Comment:
We dont want to touch this file.  The purpose of the `0.1` folder is suppose to be for a release so that we can go back and build this and that it is easy for people to inspect what is in that release.  

Can you remove touching this file?